geometry_msgs/Pose surface_pose
geometry_msgs/Pose current_pose
